File: 666-service/src/main/java/com/service/YonghuService.java
package com.service;

import java.util.List;
import pojo.TbYonghu;
import pojo.shubiao;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface YonghuService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbYonghu> findAll();
	

	//返回今日列表
	public List<TbYonghu>findtoyay();
	
	/**
	 * 增加
	*/
	public void add(TbYonghu yonghu);
	
	
	/**
	 * 修改
	 */
	public void update(TbYonghu yonghu);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbYonghu findOne(Integer id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer[] ids);

	//汇表
	public List<shubiao> biao();


}

File: 666-web/src/main/webapp/jsp/AnglaJS/js/service/yonghuService.js
//服务层
app.service('yonghuService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../yonghu/findAll');
	}
	//返回今日列表
	this.findToday=function(){
		return $http.get('../yonghu/findToday');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../yonghu/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../yonghu/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../yonghu/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../yonghu/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../yonghu/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../yonghu/search?page='+page+"&rows="+rows, searchEntity);
	}
	//查询结果
	this.count=function(){
		return $http.post('../yonghu/count');
	}

	//数表
	this.biao=function () {
		return $http.post('../yonghu/biao');
	}
});
